### PR TITLE
Additional tests for BestOfAll hint to HintHandler

### PR DESCRIPTION
**Pull request by ayobuenavista**

### CODE REVIEW

File: test/sol6/kyberHintHandler.js [hunk 1]
```diff
@@ -1657,14 +1749,37 @@ contract('KyberHintHandler', function(accounts) {
                         'reserveIds and splits must be empty'
                     );
                 });
+
+                it(`should revert the T2T hint for T2E BEST-OF-ALL HINT, E2T ${e2tTradeType} due to splits not empty`, async() => {
```
Comment (manhlx3006) [inline test/sol6/kyberHintHandler.js:1753]: Should be `T2E splits not empty`?

File: test/sol6/kyberHintHandler.js [hunk 2]
```diff
@@ -789,6 +817,26 @@ contract('KyberHintHandler', function(accounts) {
...
+                            e2tToken,
+                            e2tOpcode,
+                            e2tReserves,
+                            e2tSplits,
+                        ),
+                        'reserveIds and splits must be empty'
+                    );
+                });
+
+                it(`should revert the T2T hint for T2E BEST-OF-ALL HINT, E2T ${e2tTradeType} if splits is NOT EMPTY`, async() => {
```
Comment (manhlx3006) [inline test/sol6/kyberHintHandler.js:830]: Should be `T2E splits not empty`?

File: test/sol6/kyberHintHandler.js (hunk 1 above)
Comment (ayobuenavista) [reply test/sol6/kyberHintHandler.js]: Fixed description.

File: test/sol6/kyberHintHandler.js (hunk 2 above)
Comment (ayobuenavista) [reply test/sol6/kyberHintHandler.js]: Fixed description.

**Pull request merged**